src/decorations/functions.jl

```diff
@@ -314,8 +314,9 @@ function atan(yy::DecoratedInterval{T}, xx::DecoratedInterval{T}) where T
     # Check cases when decoration is trv and decays (from com or dac)
     if zero(T) ∈ y
         zero(T) ∈ x && return DecoratedInterval(r, trv)
-        if x.hi < zero(T) && y.lo != y.hi && signbit(y.lo) && Int(d) > 2
-            return DecoratedInterval(r, decay(d))
+        if x.hi < zero(T)
+            signbit(y.lo) && return DecoratedInterval(r, min(d, def))
```

Comment:
I think `signbit(y.lo)` will always be `false` due to the changes i have implemented. (The constructor makes that -0.0 becomes 0.0.) I'm not sure if `signbit(inf(y))` is what is needed...

Reply:
what I wanted to achieve in that line is 
```julia
y.lo < zero(T) && return DecoratedInterval(r, min(d, def))
``` 
because of the change you made that should be equivalent to `signbit(y.lo)`. Maybe it's still better to use `y.lo < zero(T)` though, clearer to read and safer to these -0 vs 0 issues.

Reply:
But it seems to work... 😄

Reply:
(side note: I think the `decay` function is not needed anymore?)

Reply:
The changes (in this PR) make that an `Interval` with upper/lower bound equal to zero, has a positive zero; so `signbit` is false *always*. Yet, `inf` returns -0.0 if the lower bound is zero, as the standard demands, in which case `signbit` return true. My point is that the `if` in line 318 is never used.

Reply:
the `if` on line 318 is used when `y.lo` is strictly lower than zero, e.g.
```julia
julia> y = DecoratedInterval(-2..0)
[-2, 0]_com

julia> x = DecoratedInterval(-2.. -1)
[-2, -1]_com

julia> atan(y, x)
[-3.1416, 3.1416]_def

```

By looking at ITL tests and playing with octave you indeed _never_ want to use that line if `y.lo == 0`